libs/ui/src/lib/graph/graph.component.ts

```diff
@@ -0,0 +1,97 @@
+
+import { AfterViewInit, ChangeDetectionStrategy, Component, ElementRef, HostListener, ViewChild } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+
+@Component({
+  selector: 'bf-graph-container',
+  templateUrl: './graph.component.html',
+  styleUrls: ['./graph.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export default class GraphContainerComponent implements AfterViewInit {
```

Comment:
to stick with what we are used to :

```suggestion
export  class GraphContainerComponent implements AfterViewInit {
```
in module
```
import { GraphContainerComponent }  from './graph.component';
```